gma/src/ios/gma_ios.mm

```diff
@@ -249,6 +249,16 @@ void OpenAdInspector(AdParent ad_parent, AdInspectorClosedListener* listener) {
   });  
 }
 
+void SetIsSameAppKeyEnabled(bool is_enabled) {
+  dispatch_async(dispatch_get_main_queue(), ^{
+    if(is_enabled) {
+      [GADMobileAds.sharedInstance.requestConfiguration setSameAppKeyEnabled:YES];
```

Comment:
There's no way to get this value (for the test)?

Reply:
Unfortunately no. There's only [a function to set the value](https://developers.google.com/admob/ios/api/reference/Classes/GADRequestConfiguration#-setsameappkeyenabled:).